docs/multicast-guide.md

```diff
@@ -137,6 +137,13 @@ Support for Windows and IPv6 will be added in the future.
 Configuration option `multicastInterfaces` is not supported with encap mode.
 Multicast packets in encap mode are SNATed and forwarded to the transport interface only.
 
+### Large number of receiver groups for one node
+
+A Linux host limits the maximum number of multicast groups it can subscribe to
+by the number of [/proc/sys/net/ipv4/igmp_max_memberships](https://sysctl-explorer.net/net/ipv4/igmp_max_memberships/).
+Users should change this number according to their needs, and Antrea is not
```

Comment:
```suggestion
Users are responsible to change the value if more than 20 multicast groups are expected per Node.
```

Reply:
updated. I think the correct wording should be "responsible for" https://www.britannica.com/dictionary/eb/qa/What-preposition-should-be-used-after-responsible-